docs/source/techspecs/object_finders.rst

```diff
@@ -239,7 +239,7 @@ the object finders in the device class (with all distractions removed):
         }
 
         sbus_device(machine_config const &mconfig, char const *tag, device_t *owner, u32 clock) :
-            device_t(mconfig, type, tag, owner, clock),
```

Comment:
Ah, so the typo was the previous line. Lemme fix that.

Reply:
No, this is supposed to be the standard designated constructor – it needs to have the exact signature `T(machine_config &, device_type, char const *, device_t *, u32)`.  If the class does not have a public constructor with that signature, it will not be well-formed.  I know the code is an over-simplification, but it should have the correct signature for this constructor and pass *something* for the device type to `device_t`.

Reply:
Oops, I don't think my brain actually registered anything AJR said. Sorry, but I guess it's ok now. Need sleep :)